src/components/message/Message.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react'
+import { UserInterface } from '~/types'
+import { Box, Avatar, IconButton, Typography } from '@mui/material'
+import { Favorite, Flag, MoreHoriz } from '@mui/icons-material'
+import { styles } from '~/components/message/Message.styles'
+
+import { spliceSx } from '~/utils/helper-functions'
+
+interface MessageProps {
+  user: UserInterface
+  timestamp?: Date | string
```

Comment:
i don't think that timestamp could be optional in message

Reply:
> i don't think that timestamp could be optional in message

hmm, seems you are right, changed that